src/explorer/SiteTreeItem.ts

```diff
@@ -124,6 +136,33 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             });
     }
 
+    private async enableScmDoBuildDuringDeploy(fsPath: string, runtime: string): Promise<void> {
+        const yesButton: MessageItem = { title: 'Yes' };
+        const dontShowAgainButton: MessageItem = { title: "Don't show again", isCloseAffordance: true };
+        const learnMoreButton: MessageItem = { title: 'Learn More' };
+        const zipIgnoreFolders: string[] = constants.getIgnoredFoldersForDeployment(runtime);
+        const buildDuringDeploy: string = `Run build script during deployment?  The "${zipIgnoreFolders.join(", ")}" directory will be built during the deployment, rather than zipped, resulting in a faster deployment.`;
```

Comment:
How about "Would you like to configure your project for faster deployment?"

I'm worried that we'll just confuse users if we include too much of the "how" in the message, and I think that's what the "learn more" button is for.